github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1293,3 +1293,64 @@ periodics:
       - name: pgp-bot-key
         secret:
           secretName: pgp-bot-key
+- name: periodic-kubevirt-e2e-k8s-1.21-sig-performance
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cron: "40 7,15,23 * * *"
+  decorate: true
+  cluster: prow-workloads
```

Comment:
What is the status of the perf cluster you have been working on? If it is ready we could include it in Prow and start running these jobs there, wdyt?

Reply:
I am investigating an [issue](https://docs.google.com/document/d/17SHOUTWF3tpOA3FhSOpz3rXia-5JgYvsX9LLi562-_0/edit#heading=h.8zwzln5oc6n) about the max number of VMIs per node in the cluster. After that, we can fully dedicate the cluster to the CI.
Then, we can update this test with CPU requirements in the new cluster.
Does it sound resonable?